client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -499,7 +503,24 @@ export const ApplicationsTable: React.FC = () => {
         ],
         getItemValue: (item) => normalizeRisk(item.risk) ?? "",
       },
+      {
+        categoryKey: "analysis",
+        title: t("terms.analysis"),
+        type: FilterType.multiselect,
+        placeholderText:
+          t("actions.filterBy", {
+            what: t("terms.analysis").toLowerCase(),
+          }) + "...",
+        selectOptions: Array.from(taskStateToAnalyze).map(
+          ([taskState, displayStatus]) => ({
+            value: taskState,
+            label: t(`${displayStatus}`),
+          })
+        ),
+        getItemValue: (item) => item?.tasks.currentAnalyzer?.state || "No Task",
```

Comment:
The value here should match the `value` prop in the  `selectOptions` list. If this would not work please let me know - there are more options.